static/_app/tools/thisistheonlylevel/index.html

```
@@ -0,0 +1,26 @@
+<html>
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)